api/src/main/java/io/strimzi/api/kafka/model/kafka/quotas/QuotasPluginStrimzi.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model.kafka.quotas;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import io.strimzi.api.kafka.model.common.Constants;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.strimzi.crdgenerator.annotations.Maximum;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.strimzi.crdgenerator.annotations.OneOf;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.util.List;
+
+@Buildable(
+    editableEnabled = false,
+    builderPackage = Constants.FABRIC8_KUBERNETES_API
+)
+@JsonInclude(JsonInclude.Include.NON_DEFAULT)
+@OneOf({
+    @OneOf.Alternative(@OneOf.Alternative.Property("minAvailableBytesPerVolume")),
+    @OneOf.Alternative(@OneOf.Alternative.Property("minAvailableRatioPerVolume"))
+})
+@JsonPropertyOrder({"type", "producerByteRate", "consumerByteRate", "minAvailableBytesPerVolume", "minAvailableRatioPerVolume", "excludedPrincipals"})
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class QuotasPluginStrimzi extends QuotasPlugin {
+
+    private static final long serialVersionUID = 1L;
+
+    public static final String TYPE_STRIMZI = "strimzi";
+
+    public static final String PLUGIN_CLASS_NAME = "io.strimzi.kafka.quotas.StaticQuotaCallback";
+
+    private Integer producerByteRate;
+    private Integer consumerByteRate;
+    private Integer minAvailableBytesPerVolume;
+    private Double minAvailableRatioPerVolume;
+    private List<String> excludedPrincipals;
+
+    @Description("Must be `" + TYPE_STRIMZI + "`")
+    @Override
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getType() {
+        return TYPE_STRIMZI;
+    }
+
+    @Description("Produce bandwidth rate quota (in bytes)")
+    @Minimum(0)
+    public Integer getProducerByteRate() {
+        return producerByteRate;
+    }
+
+    public void setProducerByteRate(Integer producerByteRate) {
+        this.producerByteRate = producerByteRate;
+    }
+
+    @Description("Consume bandwidth rate quota (in bytes)")
+    @Minimum(0)
+    public Integer getConsumerByteRate() {
+        return consumerByteRate;
+    }
```

Comment:
are these rates the same as the kafka plugin, or? should we have a longer description?

Reply:
Yeah, I think so.